package.json

```diff
@@ -59,6 +59,7 @@
     "yeoman-generator": "^5.6.1"
   },
   "resolutions": {
+    "**/sha.js": ">=2.4.12",
```

Comment:
The issue comes from crypto-browserify, which has not been updated to address this issue, hence the resolution.